cognite/client/_api_client.py

```diff
@@ -354,7 +357,10 @@ def _retrieve_multiple(
             for id_chunk in identifiers.chunked(self._RETRIEVE_LIMIT)
         ]
         tasks_summary = utils._concurrency.execute_tasks(
-            self._post, tasks, max_workers=self._config.max_workers, executor=executor
+            functools.partial(self._post, api_subversion=api_subversion),
```

Comment:
Interesting solution using `functools.partial` here 🙂

Reply:
Probably because my monster code for fetching datapoints wasn’t super easy to extend… 😂

Reply:
A simplification is on its way actually…